challenges/lib/src/main/java/challenges/stackandqueues/PseudoQueue.java

```diff
@@ -0,0 +1,25 @@
+package challenges.stackandqueues;
+
+public class PseudoQueue {

```

Comment:
I'm confused why you chose to name this front and back? These are two separate stacks.